ngsutils/bam/filter.py

```diff
@@ -700,6 +702,20 @@ def close(self):
         pass
 
 
+class MaximumMismatchRatio(object):
+    def __init__(self, ratio):
+        self.ratio = float(ratio)
+
+    def __repr__(self):
+        return "maximum mismatch ratio: %s" % self.val
```

Comment:
This line causes an error while using MaximumMismatchRation option. The problem is in `self.val` which should be `self.ratio`. Then it works fine without throwing any error.